hdlparse/verilog_parser.py

```diff
@@ -226,7 +221,7 @@ def is_verilog(fname):
     Returns:
       True when file has a Verilog extension.
     """
-    return os.path.splitext(fname)[1].lower() in ('.vlog', '.v')
+    return os.path.splitext(fname)[-1].lower() in ('.vlog', '.v', '.sv')
```

Comment:
The verilog parser currently offers next to no SystemVerilog extension support, so I'm not sure we should add `.sv` here unless we intend to start adding minimal support. The problem is even if all we did was parse `logic`, I'm pretty sure something like `wire [7:0] logic` is valid verilog... We'd ideally want to make it an optional flag. 

I am in favor of basic support as I mention here:
https://github.com/hdl/pyHDLParser/pull/4#issuecomment-972972190

but we can maybe start a different branch / PR for that?